exonum/src/helpers/fabric/details.rs

```diff
@@ -148,6 +153,121 @@ impl Command for Run {
     }
 }
 
+/// Command for running service in dev mode.
+pub struct RunDev;
+
+impl RunDev {
+    /// Returns the name of the `Run` command.
+    pub fn name() -> CommandName {
+        "run-dev"
+    }
+
+    fn artifacts_directory(ctx: &Context) -> PathBuf {
+        let directory = ctx.arg::<String>("ARTIFACTS_DIR").unwrap_or_else(|_| {
+            String::from(".exonum")
+        });
+        PathBuf::from(&directory)
+    }
+
+    fn artifacts_path(inner_path: &str, ctx: &Context) -> String {
+        let mut path = Self::artifacts_directory(ctx);
+        path.push(inner_path);
+        String::from(path.to_str().expect("Expected correct path"))
```

Comment:
supernit: The matter of personal taste, but why don't you use `into()` everywhere?

Reply:
Yeah, I see it now, that it's more favorable to use `into()` here. Ok, thanks, I'll change that )